package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "1.28.0",
+  "version": "1.29.0",
```

I think this should be:

```suggestion
  "version": "1.30.0",
```

---

updated